server/src/schema-init.ts

```diff
@@ -50,6 +50,18 @@ const runTheDB = async () => {
       rooms: [firstRoomItem._id, secondRoomItem._id],
       unreadMessages: [],
     },
+    {
+      name: {
+        first: "Vivian",
+        last: "Hu",
+      },
+      timeZone: "(UTC-07:00) Pacific Time (US & Canada)",
+      thumbnail:
+        "bc=#CBF13C;st=sq,sbc=#D061B2,sds=56%,ssx=66%,ssy=2%,or=h;st=sq,sbc=#5135F2,sds=35%,ssx=83%,ssy=33%,or=v;st=rt,sbc=#088395,sds=31%,ssx=17%,ssy=40%,or=h",
+      workingHours: [],
+      rooms: [firstRoomItem._id, secondRoomItem._id],
+      unreadMessages: [],
```

Comment:
I know we really shouldn't have to do it this way. Let's see if we can find a time to chat about how to remove this.